.ado/jobs/sample-apps.yml

```diff
@@ -64,19 +64,39 @@ jobs:
           verbosityRestore: Detailed # Options: quiet, normal, detailed
         condition: succeeded()
 
+      - task: PowerShell@2
```

Comment:
This seems to share a lot between playground.yml..

Okay for now, when we do a 3rd we should consider if we can split this out into a seperate parameterized yaml file and call it from both locations.

Reply:
Sure I can definitely do that in a second wave of changes.